File: demo/java-demo/src/test/java/com/alibaba/demo/one2multi/ASvcMock.java
package com.alibaba.demo.one2multi;

import com.alibaba.testable.core.annotation.MockInvoke;

public class ASvcMock {

    @MockInvoke(targetClass = String.class, targetMethod = "format")
    public String a_format(String format, Object... args) {
        return "a_mock";
    }

}

File: demo/kotlin-demo/src/test/kotlin/com/alibaba/demo/one2multi/ASvcMock.kt
package com.alibaba.demo.one2multi

import com.alibaba.testable.core.annotation.MockInvoke

class ASvcMock {

    @MockInvoke(targetClass = String::class, targetMethod = "format")
    fun a_format(format: String, vararg args: Any?): String {
        return "a_mock"
    }

}
